zerodb/models/__init__.py

```diff
@@ -1,12 +1,28 @@
 import persistent
 import fields
 import exceptions
+from collective.indexing.queue import getQueue
+from collective.indexing.subscribers import objectModified
+from zope.lifecycleevent.interfaces import IObjectModifiedEvent
+from zope.lifecycleevent import ObjectModifiedEvent
+from zope.lifecycleevent import modified
+from zope.event import subscribers
 from zerodb.intid import IdStore
 from zerodb.trees import family32
 from zerodb.catalog import Catalog
 from zerodb.models.fields import Field, Text
 
 
+def zerodb_autoreindex_dispatcher(event):
+    if isinstance(event, ObjectModifiedEvent):
+        objectAutoReindex(event)
```

Comment:
collective.indexing has an [optimize](https://github.com/plone/collective.indexing/blob/master/src/collective/indexing/queue.py#L151) before process queue. The optimize seems could [reduce repeated reindex obj](https://github.com/plone/collective.indexing/blob/master/src/collective/indexing/queue.py#L122)